modules/dnn/test/test_onnx_importer.cpp

```diff
@@ -0,0 +1,119 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+// Copyright (C) 2018, Intel Corporation, all rights reserved.
+// Third party copyrights are property of their respective owners.
+
+
+#include "test_precomp.hpp"
+#include "npy_blob.hpp"
+#include <opencv2/dnn/shape_utils.hpp>
+
+namespace opencv_test { namespace {
+
+template<typename TString>
+static std::string _tf(TString filename)
+{
+    String rootFolder = "dnn/onnx/";
+    return findDataFile(rootFolder + filename, false);
+}
+
+class Test_ONNX_layers : public DNNTestLayer
+{
+public:
+    void testLayerUsingONNXModels(const String& basename)
+    {
+        String modelsFolder = "models/";
+        String onnxmodel = _tf(modelsFolder + basename + ".onnx");
```

Comment:
I think we can remove `modelsFolder` variable:
String onnxmodel = _tf("models/"+ basename + ".onnx");